### DISCUSSION TITLE
Why does Cora training set + validation Set + test set only use 1640 nodes instead of 2708？

### DISCUSSION THREAD

**Discussion by Chrisable** [pyg-team/pytorch_geometric#4627] [General]
In general,the number of the training nodes is 140,the validation Set is 500,the  test nodes is 1000.why is not the sum 2708?When only using 1640 nodes,what is the benefits?

**Comment by Padarn** [comment]
Cora is implemented in a number of different datasets. Can you be more specific?

**Comment by rusty1s** [comment]
This is just a pre-defined split introduced in the GCN paper, where we only want to train on 20 examples per class. I don't think there is a real reason to not utilize all node labels during testing, but it is what it is :)
